src/stylesheets/elements/_headings.scss

```diff
@@ -147,6 +135,15 @@ footer h2 {
   }
 }
 
+.hero h1 {
+  @include media($tablet) {
```

Comment:
Do you want to `@include line-length` here?

Reply:
No, different values, at different break-points

Reply:
My bad...